docs/layers.md

```diff
@@ -500,10 +498,10 @@ Features from OpenStreetMap which are tagged `disused=*` for any other value tha
 * `access`: Whether the bicyle parking is for general public use (`yes`, `permissive`, `public`) or for customers only (`customers`) or private use only (`private`, `no`).
 * `capacity`: Approximate number of total bicycle parking spots.
 * `covered`: Is the parking area covered.
-* `fee`: Whether a fee must be paid to use the bicycle parking (`yes`/`no`).
+* `fee`: Whether a fee must be paid to use the bicycle parking (if `true`, the default is `false`).
```

Comment:
We do export `false` for "fee" - as in [this test case](https://github.com/mapzen/vector-datasource/blob/master/test/592-add-adjust-bicycle-pois.py#L97-L100). I've updated the docs in #825, which hopefully make that clearer.